decidim-core/config/locales/en.yml

```diff
@@ -96,23 +97,26 @@ en:
         other: about %{count} months
       half_a_minute: half a minute
       less_than_x_minutes:
-        one: less than a min.
-        other: less than %{count} min.
+        one: less than a minute
+        other: less than %{count} minutes
       less_than_x_seconds:
         one: right now
-        other: less than %{count} sec.
+        other: less than %{count} seconds
       x_days:
-        one: 1 day ago
-        other: "%{count} days ago"
+        one: 1 day
+        other: "%{count} days"
       x_hours:
-        one: 1 hour ago
-        other: "%{count} hours ago"
+        one: 1 hour
+        other: "%{count} hours"
       x_minutes:
-        one: 1 min. ago
-        other: "%{count} min. ago"
+        one: 1 minute
+        other: "%{count} minutes"
+      x_months:
+        one: 1 month
+        other: "%{count} months"
```

Comment:
I believe the months translation is not used anywhere, so do we really need to add it?

Looking at the code where this is used, the `time_ago_in_words` is only used when the time is between now and 1 month ago:
https://github.com/decidim/decidim/blob/75d19a7bec8a759293e0804cbdd5151b50b01209/decidim-core/app/presenters/decidim/notification_presenter.rb#L13-L14

Reply:
actually, the month ago is may be used if user has messages older than 30 days: 

https://github.com/decidim/decidim/blob/ff355645ca38e3d32cae447f5084120def4b3160/decidim-core/app/cells/decidim/user_conversations/conversation_item.erb#L22

Reply:
Ah, great! Then adding it is considered a bugfix and therefore totally fine.